vue-app/src/api/core.ts

```diff
@@ -1,7 +1,22 @@
 import { ethers } from 'ethers'
+import WalletConnectProvider from '@walletconnect/web3-provider'
 
 import { FundingRoundFactory } from './abi'
 
+export const providers = {
+  metamaskProvider: new ethers.providers.StaticJsonRpcProvider(
+    process.env.VUE_APP_ETHEREUM_API_URL
+  ),
+  walletconnectProvider: new WalletConnectProvider({
+    infuraId: '4299898b85054e0386e7136ab3170d11',
+    rpc: {
+      // 1: "http://localhost:18545",
+      3: 'https://rinkeby.infura.io/v3/',
+    },
+    // chainId: 1
```

Comment:
Shouldn't we be using env variables here? (perhaps you've already mentioned something about this, sorry if I'm out of context)